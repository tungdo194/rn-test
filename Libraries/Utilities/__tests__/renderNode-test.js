/**
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @format
 * @emails oncall+react_native
 */

'use strict';

const React = require('react');
const ReactTestRenderer = require('react-test-renderer');
const renderNode = require('../renderNode');

describe('renderNode', () => {
  it('renders simple component', () => {
    class Test extends React.Component<{||}> {
      render() {
        return null;
      }
    }

    const component = ReactTestRenderer.create(renderNode(Test));
    expect(component).toMatchSnapshot();
  });
});
