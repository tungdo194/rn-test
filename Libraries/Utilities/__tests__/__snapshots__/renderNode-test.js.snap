// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderNode renders simple component 1`] = `null`;
